config/locales/en.yml

```diff
@@ -217,6 +224,14 @@ en:
         list_1: We do not save your online banking username and password
         list_2: We do not have any ongoing access to your bank accounts
         list_3: We only use your details to assess your eligibility for legal aid
+    own_homes:
+      show:
+        h1-heading: Do you own the home that you live in?
+        own_home:
+          mortgage: Yes, with a mortage or loan
+          owned_outright: Yes, owned outright
+          not: No
```

Comment:
in the view this shows up as 'Own home no'.
it might be possible to use the generic no in the en.yml file for this radio button